content/updates/2019-09-25-v12-is-here.md

```diff
@@ -0,0 +1,132 @@
++++
+date = "2019-09-25"
+title = "v12 Is Here" 
++++
+
+The end of the quarter brings a lot of new updates to cloud.gov! Normally, when we upgrade the platform, it's all very discreet and we don't discuss it externally unless asked.
+
+As part of our commitment to being better, we feel it's time to start highlighting some of what we do. It is well known that cloud.gov runs entirely on open source software, composing various bits of public software from the Cloud Foundry ecosystem.
+
+The Cloud Foundry ecosystem publishes what is known as [`cf-deployment`](https://github.com/cloudfoundry/cf-deployment), a canonical reference for how to deploy the platform. cloud.gov consumes the base reference, applies government-specific security policies and procedures, and deploys it. Continuously. In the last quarter, we've continuously deployed the platform **about 150 times**. Most of these fixes are patches, security updates, or configuration changes.
+
+This month there was a major release of the canonical deployment reference, version v12.0.0, which means we consume it, it's updates, apply our changes, and deploy it. As of writing this, v12 is deployed and available to customers. While some changes highlighted are designed for the cloud.gov operations team, there are 50 CVEs patched, so it's worth talking about.
+
+You'll find a human-friendly description and how it's relevant to major components of the system.
+
+## Identity
+
+There were some discreet features designed for operators, but there were two high scored security patches fixed. Customers are no longer vulnerable to these CVEs when authenticating with the platform.
+
+* [CVE-2019-11279](https://nvd.nist.gov/vuln/detail/CVE-2019-11279)
+* [CVE-2019-11278](https://nvd.nist.gov/vuln/detail/CVE-2019-11278)
+
+## Buildpacks
+
+### Java Buildpack
+
+This release bumps the Java buildpack from v4.20 to v4.21.
+
+* Metric Writer Augments with Cloud Foundry-specific dimensions (via [many people](https://github.com/cloudfoundry/java-buildpack/issues/644))
+* Update Introscope Agent Version (via [Dhruv Mevada](https://github.com/cloudfoundry/java-buildpack/pull/739))
+* Shade Auto-reconfiguration Jackson Dependency (via [@Agraham21](https://github.com/cloudfoundry/java-buildpack-auto-reconfiguration/issues/69) and [@pborbas](https://github.com/cloudfoundry/java-buildpack/issues/742))
+
+## Containers
+
+This section highlights a component called Diego, which is the underpinning container orchestration technology of cloud.gov. Most of these fixes are never going to be seen by customers, but are important to the health and stability of the cloud.gov platform. For the interested, feel free to do a [deep dive on the container orchestrator](https://github.com/cloudfoundry/diego-design-notes).
```

Comment:
Change "Most of these fixes are never going to be seen by customers" to "Although most of these fixes are not customer-facing, they are important to the health and stability of the cloud.gov platform."